junit-platform-console/src/main/java/org/junit/platform/console/options/SelectorConverter.java

```diff
@@ -109,4 +120,12 @@ public DiscoverySelectorIdentifier convert(String value) {
 		}
 	}
 
+	private static URI stripQueryComponent(URI uri) {
```

Comment:
This method is a duplicate of the one in `org.junit.platform.engine.support.descriptor.ResourceUtils`, but I don't think there is a way of using the same in both places, is it?

Reply:
If we made `ResourceUtils` and the method public, we could reuse it. I think it would be slightly better regarding consistenct to move it to the `org.junit.platform.commons.util` package, though.